distribution/scripting.md

```diff
@@ -8,14 +8,18 @@ Each script is a single physical javascript file within the `plugin` directory i
 - Mac: `/Users/YourName/Library/Application Support/OpenRCT2`
 - Linux: `$XDG_CONFIG_HOME/OpenRCT2` or in its absence `$HOME/.config/OpenRCT2`
 
+You can also find the user directory via the “Open custom content folder” option under the red toolbox button in the game's title screen.
+
 OpenRCT2 will load every single file with the extension `.js` in this directory recursively. So if you want to prevent a plug-in from being used, you must move it outside this directory, or rename it so the filename does not end with `.js`.
 
+## Script types
+
 There are three types of scripts:
 * Local
 * Remote
 * Intransient
 
-Local scripts can **not** alter the game state. This allows each player to enable any local script for their own game without other players needing to also enable the same script. These scripts tend to provide extra tools for productivity, or new windows containing information.
+Local scripts **cannot** alter the game state directly. This allows each player to enable any local script for their own game without other players in a multiplayer server also needing to also enable the same script. Local scripts **can** interact with the game by mimicking a player's actions through the built-in “game actions”. These scripts tend to provide extra tools for productivity, or new windows containing information.
```

Comment:
The section "What does the error 'Game state is not mutable in this context' mean?" is similarly wrong.

Reply:
Yes, I can clarify the difference between singleplayer and multiplayer as you mentioned, as local vs. remote has no differences there. 

For multiplayer I do think "remote" by default is better. It's better that the plugin you bring doesn't load at all than that it loads but breaks/errors because the plugin developer didn't take mutable context in account. Having it as "remote" by default is more intuitive for users and developers to understand why it doesn't load in multiplayer if multiplayer hasn't been accounted for, as it's "oh I need to do something extra for multiplayer" (as you explained).

Note that remove server plugins can exist without custom game actions; e.g. UI popups/welcome messages when a player joins, spam protection, tools to kick/ban players, block certain built-in game actions by subscribing to the hooks, run certain other code tied to hooks ("interval.day", "ratings.calculate" etc.).

Reply:
I think the main point of the local vs remote question is "What does my plugin want to do?" and not "Should my plugin target singleplayer or multiplayer?" and I think in the past there was a lot of confusion about this, maybe because the naming could suggest that "local" is for single player and "remote" is for multiplayer, which is absolutely not the case.